addOns/pscanrulesBeta/src/test/java/org/zaproxy/zap/extension/pscanrulesBeta/SiteIsolationScanRuleTest.java

```diff
@@ -340,6 +341,30 @@ void shouldRaiseAlertGivenCoepHeaderIsNotEqualsToRequireCorp() throws Exception
         assertThat(alertsRaised.get(0).getEvidence(), equalTo("unsafe-none"));
     }
 
+    @ParameterizedTest
+    @ValueSource(strings = {"require-corp", "credentialless"})
+    void shouldNotRaiseAlertGivenCoepHeaderIsAnExpectedValue(String directive) throws Exception {
+        // Ref: https://html.spec.whatwg.org/multipage/origin.html#the-headers
+        // Given
+        HttpMessage msg = new HttpMessage();
+        msg.setRequestHeader("GET / HTTP/1.1");
+        msg.setResponseHeader(
+                "HTTP/1.1 200 OK\r\n"
+                        + "Content-Type: text/html; charset=iso-8859-1\r\n"
+                        + "Cross-Origin-Resource-Policy: same-origin\r\n"
+                        + "Cross-Origin-Embedder-Policy: "
+                        + directive
+                        + "\r\n"
+                        + "Cross-Origin-Opener-Policy: same-origin\r\n");
```

Comment:
It would be better to extract a method since this is exactly the same as for `shouldRaiseAlertGivenCoepHeaderIsNotExpectedValue`.

Reply:
Can also use text blocks.